lops/lop-load.dts

```diff
@@ -34,5 +34,10 @@
                         compatible = "system-device-tree-v1,lop,load";
                         load = "assists/openamp-xlnx.py";
                 };
+                lop_3 {
+                        compatible = "system-device-tree-v1,lop,load";
+                        load = "assists/openamp-xlnx-zynq.py";
```

Comment:
I may have mislead you in our conversations earlier .. because I didn't quite grok what you were asking.

What is the delta of this assist versus the generic one ? It would be preferable if we have one assist, but have two different lop files.

We can pass an argument in the assist call, that the single assist reads and performs zynq versus versal differences, etc.

What we have here, looks like a lot of duplicated logic.

Reply:
Hi Bruce,

right in the pull request I plan to clean up as follows for that:
1 assist for memor carveouts
1 assist for ipi handling
1 assist for generating the outfile

given these 3 we can plug and play as needed for the zynq zynqmp or versal cases as warranted

Reply:
ok. so the summary is .. there's a lot of duplicated code here, but we'll remove that duplication by splitting the processing further in the future.

Reply:
yes exactly